### PR TITLE
Add test to comment counter

### PR DESCRIPTION
**Pull request by yayner2002**
In this PR I have implemented the following features

- Test the counter function using Jest and used three test data as

1. Check whether the returned array of object's length is the same as expected
2. check whether the array given's length is the same as expected
3. and check the returned value for invalid data is sam as expected

### CODE REVIEW

**Review by medsonmoombe — APPROVED** [review]
## Status Approved :+1: 
good job

- Hello @yayner2002  Great job :100:  Implementing all the tasks requirement, Your work is done all the tests have passed. There is nothing else to say , its time to merge it :+1: 


- However i  have some optional suggestions to make your code look more professional , kindly check my inline optional suggestions down

**Comment by Epaltechs** [comment]
I applaud the nice job done @yayner2002. image
Please correct the typo in the test folder to 'properly' instead of 'properlly'.  Well done! image

File: src/modules/Involvement.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// const URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+// const appID = 'xKdCcCB1CZVZ3fVJxMXD';
+
+// const postLike = async (itemID) => {
+//     const response = await fetch(`${URL}${appID}/likes`, {
+//       method: 'POST',
+//       headers: {
+//         'Content-Type': 'application/json',
+//       },
```
Comment (medsonmoombe) [inline src/modules/Involvement.js:9]: please Kindly consider removing unused code in the code base,  unused code makes the code look unprofessional.

File: src/modules/countNumOfComments.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// import { countNumOfComments } from './commentsItemDetail.js';
```
Comment (medsonmoombe) [inline src/modules/countNumOfComments.test.js:1]: Please consider removing unused code in the code, unused code makes your work look unprofessional

**Pull request merged**